src/main/java/lifeline/Lifeline.java

```diff
@@ -0,0 +1,102 @@
+package lifeline;
+
+import java.util.ArrayList;
+
+import lifeline.command.Command;
+import lifeline.exception.LifelineException;
+import lifeline.parser.Parser;
+import lifeline.storage.Storage;
+import lifeline.task.TaskList;
+import lifeline.ui.Ui;
+
+/**
+ * The class Lifeline is the entry point to the Lifeline program.
+ * Calling start on this class will start the program.
+ */
+public class Lifeline {
+
+    /**
+     * List of tasks created by the user
+     */
+    private TaskList taskList;
+
+    /**
+     * Used to save new tasks or load saved tasks
+     */
+    private Storage storage;
+
+    /**
+     * Used to display information to the user
+     */
+    private Ui ui;
+
+    /**
+     * Default constructor for a Lifeline
+     *
+     * @param filepath Path to load or save tasks
+     */
+    public Lifeline(String filepath) {
+        this.storage = new Storage(filepath);
+        this.ui = new Ui();
+        try {
+            this.taskList = storage.load();
+        } catch (LifelineException e) {
+            this.taskList = new TaskList(new ArrayList<>());
+        }
+    }
+
+    /**
+     * Gets the appropriate response from Lifeline if user inputs a valid command
+     * Method is used only for GUI
+     *
+     * @param command User input
```

Comment:
Perhaps could put punctuation behind the parameter descriptions as mentioned in the coding standards doc.

Reply:
I completely missed that. Thanks!